### PR TITLE
A man a plan a savas

### PR DESCRIPTION
**Pull request by chrisarusso**
The build is failing due to a link that is failing in master. 

@kostajh please have a look at the post. In addition to the post, I've updated the tags file comment to help instruct better.

### CODE REVIEW

File: _posts/2015-06-18-a-man-a-plan-a-canal-a-savas.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+layout: post
+title: "A man a plan a canal a... Savas!"
+date: 2015-06-18
+author: Chris Russo
+tags: business identity
+summary: The truth behind the palindrome.
+---
+
+Previously, on our [who we are](/team) page, I gave you a taste of the 
+[meaning](/team#what-does-savas-mean) of Savas. Now let's take a deeper dive
```
Comment (AnneTee) [inline _posts/2015-06-18-a-man-a-plan-a-canal-a-savas.md]: Need some punctuation at the end of this line. Might I recommend a few exclamation points?!?!

File: _posts/2015-06-18-a-man-a-plan-a-canal-a-savas.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+For those who know me, they _think_ I'm zany. For those who know me well, they
+**know** I'm zany. For the select few who know me really well, and/or have worked 
+with me, they know I have an odd obsession with 
+[palindromes](http://en.wikipedia.org/wiki/Palindrome).
+
+This is why, when we asked [Harbor](http://www.studioharbor.com) to help us
+come up with a company identity, including designing our logo and our name, 
+ "palindrome" was naturally a category... e pluribus unum ;)
```
Comment (AnneTee) [inline _posts/2015-06-18-a-man-a-plan-a-canal-a-savas.md]: The winky face makes me have a frowny face IRL

**Comment by AnneTee** [comment]
@chrisarusso I'm also unclear on what the "canal" part of the title means :boat:

**Comment by chrisarusso** [comment]
@AnneTee `a man a plan a canal, panama!` is a (fairly) **well-known** palindrome

**Comment by AnneTee** [comment]
@chrisarusso Ohhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh :bulb:

**Pull request merged**